.travis.yml

```diff
@@ -6,9 +6,9 @@ matrix:
       jdk: oraclejdk9
     - os: osx
 
-script: >-
-    ./config/travis/run-checks.sh &&
-    time travis_retry ./gradlew clean checkstyleMain checkstyleTest test functional
+script:
+  - ./config/travis/run-checks.sh
+  - time travis_retry ./gradlew clean checkstyleMain checkstyleTest test functional
```

Comment:
if you change this, I think it will run travis time even if run-checks.sh had failed.